Makefile

```diff
@@ -9,6 +9,9 @@ LAST_APACHE_BASE_PATH := $(shell if [ -f .build-artefacts/last-apache-base-path
 API_URL ?= //mf-chsdi3.dev.bgdi.ch
 LAST_API_URL := $(shell if [ -f .build-artefacts/last-api-url ]; then cat .build-artefacts/last-api-url 2> /dev/null; else echo '-none-'; fi)
 PUBLIC_URL ?= //public.dev.bgdi.ch
+PUBLIC_ALLOWED_URL_REGEXP ?= https?:\/\/public\..*(\.admin\.ch|\.bgdi\.ch)\/.*
+PUBLIC_URL_REGEXP ?= ^https?:\/\/public\..*(bgdi|geo\.admin)\.ch.*
```

Comment:
`PUBLIC_ALLOWED_URL_REGEXP`  and `PUBLIC_URL_REGEXP` do the same thing . Can you remove `PUBLIC_ALLOWED_URL_REGEXP` pls ?